packages/engine/src/scene/functions/loadGLTFModel.ts

```diff
@@ -301,6 +302,9 @@ export const generateEntityJsonFromObject = (rootEntity: Entity, obj: Object3D,
   if (skinnedMesh.isSkinnedMesh) setComponent(objEntity, SkinnedMeshComponent, skinnedMesh)
   else setComponent(objEntity, FrustumCullCameraComponent)
 
+  if (mesh.isMesh && !instancedMesh.isInstancedMesh && !skinnedMesh.isSkinnedMesh)
+    setComponent(objEntity, MeshBVHComponent, { mesh: mesh })
```

Comment:
this is going to add a BVH for every object in the model's heirarchy. we don't want that, just one for each model (or lone mesh)

we need to use useMeshOrModel for the visualizer